### ISSUE TITLE
After snap autorefresh to 18.0.4snap1 nextcloud inaccessable. Reports internal server error.

### ISSUE THREAD

**Issue by shaneburrell** [nextcloud/nextcloud-snap#1318]
#### Describe the bug
After snap autorefresh to 18.0.4snap1 nextcloud inaccessable. Reports internal server error.

#### To Reproduce
Immedialy after snap auto refresh. Nextcloud is no longer accessable.

#### Expected behavior
Internal Server Error
The server was unable to complete your request.

If this happens again, please send the technical details below to the server administrator.

More details can be found in the server log.

#### OS/snapd/snap version

    $ snap list nextcloud
Name       Version      Rev    Tracking       Publisher   Notes
nextcloud  18.0.4snap1  20498  latest/stable  nextcloud✓

and:

    $ snap version
snap    2.44.3
snapd   2.44.3
series  16
ubuntu  18.04
kernel  4.15.0-1065-aws

#### Logs
 <summary>Apache</summary>
 <p>

### Status
```
● snap.nextcloud.php-fpm.service - Service for snap application nextcloud.php-fpm
   Loaded: loaded (/etc/systemd/system/snap.nextcloud.php-fpm.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2020-04-25 15:27:11 UTC; 15min ago
 Main PID: 1650 (start-php-fpm)
    Tasks: 2 (limit: 4915)
   CGroup: /system.slice/snap.nextcloud.php-fpm.service
           ├─1650 /bin/sh /snap/nextcloud/20498/bin/start-php-fpm
           └─2372 php-fpm: master process (/snap/nextcloud/20498/config/php/php-fpm.conf)

Apr 25 15:27:11 ip-172-26-0-132 systemd[1]: Started Service for snap application nextcloud.php-fpm.
Apr 25 15:27:12 ip-172-26-0-132 nextcloud.php-fpm[1650]: Waiting for MySQL...
Apr 25 15:27:16 ip-172-26-0-132 nextcloud.php-fpm[1650]: done
Apr 25 15:27:16 ip-172-26-0-132 nextcloud.php-fpm[1650]: Obtaining nextcloud mysql credentials... done
```
### Journal
```
-- Logs begin at Tue 2020-03-17 02:25:58 UTC, end at Sat 2020-04-25 15:42:27 UTC. --
Mar 17 02:33:13 ip-172-26-15-114 systemd[1]: Started Service for snap application nextcloud.php-fpm.
Mar 17 02:33:13 ip-172-26-15-114 nextcloud.php-fpm[32101]: Waiting for MySQL...
Mar 17 02:33:24 ip-172-26-15-114 nextcloud.php-fpm[32101]: done
Mar 17 02:33:24 ip-172-26-15-114 nextcloud.php-fpm[32101]: Obtaining nextcloud mysql credentials... done
Mar 30 19:12:30 ip-172-26-15-114 systemd[1]: Stopping Service for snap application nextcloud.php-fpm...
Mar 30 19:12:30 ip-172-26-15-114 systemd[1]: Stopped Service for snap application nextcloud.php-fpm.
Mar 30 19:12:41 ip-172-26-15-114 systemd[1]: Started Service for snap application nextcloud.php-fpm.
Mar 30 19:12:42 ip-172-26-15-114 nextcloud.php-fpm[21411]: Waiting for MySQL...
Mar 30 19:12:45 ip-172-26-15-114 nextcloud.php-fpm[21411]: done
Mar 30 19:12:45 ip-172-26-15-114 nextcloud.php-fpm[21411]: Obtaining nextcloud mysql credentials... done
-- Reboot --
Apr 25 15:22:26 ip-172-26-0-132 systemd[1]: Started Service for snap application nextcloud.php-fpm.
Apr 25 15:22:33 ip-172-26-0-132 nextcloud.php-fpm[1146]: Waiting for MySQL...
Apr 25 15:22:49 ip-172-26-0-132 nextcloud.php-fpm[1146]: done
Apr 25 15:22:49 ip-172-26-0-132 nextcloud.php-fpm[1146]: Obtaining nextcloud mysql credentials... done
Apr 25 15:24:54 ip-172-26-0-132 systemd[1]: Stopping Service for snap application nextcloud.php-fpm...
Apr 25 15:24:54 ip-172-26-0-132 systemd[1]: Stopped Service for snap application nextcloud.php-fpm.
-- Reboot --
Apr 25 15:27:11 ip-172-26-0-132 systemd[1]: Started Service for snap application nextcloud.php-fpm.
Apr 25 15:27:12 ip-172-26-0-132 nextcloud.php-fpm[1650]: Waiting for MySQL...
Apr 25 15:27:16 ip-172-26-0-132 nextcloud.php-fpm[1650]: done
Apr 25 15:27:16 ip-172-26-0-132 nextcloud.php-fpm[1650]: Obtaining nextcloud mysql credentials... done
```
 </p>
</details>

<details>
 <summary>mDNS Publisher</summary>
 <p>

<details>
 <summary>Certificate Renewer</summary>
 <p>

<details>
 <summary>Nextcloud Cron</summary>
 <p>

<details>
 <summary>Nextcloud Fixer</summary>
 <p>

<details>
 <summary>Nextcloud configuration</summary>
 <p>

```
 </p>
</details>

<details>
 <summary>Nextcloud Fixer</summary>
 <p>

<details>
 <summary>Nextcloud configuration</summary>
 <p>

```
 </p>
</details>

<details>
 <summary>snap version</summary>
 <p>

```
snap    2.44.3
snapd   2.44.3
series  16
ubuntu  18.04
kernel  4.15.0-1065-aws
```
 </p>
</details>

<details>
 <summary>nextcloud version</summary>
 <p>

```
Name       Version      Rev    Tracking       Publisher   Notes
nextcloud  17.0.5snap1  20007  latest/stable  nextcloud*  disabled
nextcloud  18.0.4snap1  20498  latest/stable  nextcloud*  -
```
 </p>
</details>

**Comment by shaneburrell** [comment]
I've got it working again. Something went wrong with upgrade related to Zimbra drive.  

An unhandled exception has been thrown:
OCP\AutoloadNotAllowedException: Autoload path not allowed: /var/snap/nextcloud/20498/nextcloud/extra-apps/zimbradrive/lib/auth/
zimbrausersbackend.php in /snap/nextcloud/20498/htdocs/lib/autoloader.php:139

I removed the directory.  I was able to get past the error and taken out of maintenance mode.
